wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationController.swift

```diff
@@ -474,23 +482,41 @@ extension ConversationCreationController {
         present(alertViewController, animated: true)
     }
 
-    func encryptionProtocolPicker(_ completion: @escaping (EncryptionProtocol) -> Void) -> UIAlertController {
-        let alert = UIAlertController(title: L10n.Localizable.Conversation.Create.Mls.pickerTitle, message: nil, preferredStyle: .actionSheet)
+    func encryptionProtocolPicker(_ completion: @escaping (Feature.MLS.Config.MessageProtocol) -> Void) -> UIAlertController {
+        typealias Localizable = L10n.Localizable.Conversation.Create
 
-        for encryptionProtocol in EncryptionProtocol.allCases {
-            alert.addAction(UIAlertAction(title: encryptionProtocol.rawValue, style: .default, handler: { _ in
-                completion(encryptionProtocol)
-            }))
-        }
+        let mlsFeature = userSession.makeGetMLSFeatureUseCase().invoke()
+        let proteus = mlsFeature.config.defaultProtocol == .proteus ? Localizable.ProtocolSelection.proteusDefault : Localizable.ProtocolSelection.proteus
+        let mls = mlsFeature.config.defaultProtocol == .mls ? Localizable.ProtocolSelection.mlsDefault : Localizable.ProtocolSelection.mls
 
-        alert.popoverPresentationController?.permittedArrowDirections = [ .up, .down ]
-        alert.addAction(UIAlertAction(title: L10n.Localizable.Conversation.Create.Mls.cancel, style: .cancel, handler: nil))
+        let alert = UIAlertController(
+            title: Localizable.Mls.pickerTitle,
+            message: nil,
+            preferredStyle: .actionSheet
+        )
+        alert.addAction(UIAlertAction(
+            title: proteus,
+            style: .default,
+            handler: { _ in
+                completion(.proteus)
+            }
+        ))
+        alert.addAction(UIAlertAction(
+            title: mls,
+            style: .default,
+            handler: { _ in
+                completion(.mls)
+            }
+        ))
+        alert.addAction(UIAlertAction(
+            title: Localizable.Mls.cancel,
+            style: .cancel
+        ))
+        alert.popoverPresentationController?.permittedArrowDirections = [
+            .up,
+            .down
+        ]
 
         return alert
     }
 }
-
-enum EncryptionProtocol: String, CaseIterable {
-    case proteus = "Proteus (default)"
-    case mls = "MLS"
-}
```

Comment:
This was replaced by `Feature.MLS.Config.MessageProtocol` and localised strings.